### ISSUE TITLE
csv use

### ISSUE THREAD

**Issue by nzyeehaa** [GiovanniSalmeri/yellow-openstreetmap#4]
@GiovanniSalmeri
great extension, and good work on using OSM. 
you said the csv is configurable...

can the map use the csv to plot all the markers?
can the map be centered at a certain address and show all csv markers around it?
can text be added to a marker?

**Comment by GiovanniSalmeri** [comment]
Evidently I completely forgot your question, I am very sorry! When I wrote the extension I tried to implement only the basic features: so not different markers. As soon as possible I will see if the OSM API allows to do easily what you asked for.

**Comment by GiovanniSalmeri** [comment]
In order to obtain what you would like, I think that the best thing would be another extension using [Leaflet](http://leafletjs.com/). There [used to be one](https://github.com/nibreh/yellow-plugin-leaflet), but now it would require a complete rewrite. If @nibreh agrees I can put it in my TODO list.

**Comment by GiovanniSalmeri** [comment]
@nzyeehaa With the latest update, all that you asked for is now possible, thanks to [Leaflet](https://leafletjs.com/). This library is loaded only if needed: therefore the extension is light as before if used for simple maps without points of interests.

**Comment by nzyeehaa** [comment]
@GiovanniSalmeri thanks so much for your brilliant work, i havent had a chance to test it out yet. its also a shame this isnt in the yellow directory anymore

**Comment by nzyeehaa** [comment]
@GiovanniSalmeri I have finally come back to this extension for possible use on a new personal project.
I installed a new yellow 0.9 then Openstreetmap 0.9.1

It appears that it wont work on a local host, so I re-set it up online.
I can get the map working fine with one marker, however the csv seems to have some trouble.

the file is located at media/openstreetmap
I used the data as per the example page in a csv, and when that failed - I trimmed it to only one entry, when that failed, I trimmed it to one word ( tried with and without trailing comma ) 

each time using the csv file I get a console error

`Uncaught SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at openstreetmap.js:34:36
    at NodeList.forEach (<anonymous>)
    at HTMLScriptElement.<anonymous> (openstreetmap.js:32:18)`

I am picking it is possibly due to updates?
